pkg/operator/staticpod/controller/node/node_controller.go

```diff
@@ -79,18 +79,7 @@ func (c *NodeController) sync(ctx context.Context, syncCtx factory.SyncContext)
 			}
 		}
 		if found {
-			newTargetNodeState := applyoperatorv1.NodeStatus().
-				WithNodeName(originalOperatorStatus.NodeStatuses[i].NodeName).
-				WithCurrentRevision(originalOperatorStatus.NodeStatuses[i].CurrentRevision).
-				WithTargetRevision(originalOperatorStatus.NodeStatuses[i].TargetRevision).
-				WithLastFailedRevision(originalOperatorStatus.NodeStatuses[i].LastFailedRevision).
-				WithLastFailedReason(originalOperatorStatus.NodeStatuses[i].LastFailedReason).
-				WithLastFailedCount(originalOperatorStatus.NodeStatuses[i].LastFailedCount).
-				WithLastFallbackCount(originalOperatorStatus.NodeStatuses[i].LastFallbackCount).
-				WithLastFailedRevisionErrors(originalOperatorStatus.NodeStatuses[i].LastFailedRevisionErrors...)
-			if originalOperatorStatus.NodeStatuses[i].LastFailedTime != nil {
-				newTargetNodeState = newTargetNodeState.WithLastFailedTime(*originalOperatorStatus.NodeStatuses[i].LastFailedTime)
-			}
+			newTargetNodeState := applyoperatorv1.NodeStatus().WithNodeName(originalOperatorStatus.NodeStatuses[i].NodeName)
```

Comment:
i'm glad that you found the issue, thank you so much!

this change looks correct to me based on the info from the docs:
```
 If set or map, different managers can manage entries separately.
 ```